application/web/main.css

```diff
@@ -0,0 +1,50 @@
+:root {
+    --journal-entry: #F3F3F3;
+    --sliding-shelf: #EBEBEB;
+    --main-stroke-color: #A1A1A1;
+    --grey-text-color: #404040;
+    --icon-color: #D7D7D7;
```

Comment:
The design didn't end up using the icon-color so its okay to delete this styling.